Tests/XCUIElementWrapper_Tests.swift

```diff
@@ -57,6 +57,15 @@ class XCUIElementWrapper_Tests: SwiftUITestCase {
         XCTAssertTrue(ele.isEnabled)
     }
 
+    func test_pick() {
+        app.find(label: "Picker Views").tap()
+        app.find(label: "Counter").tap()
+        app.find(value: "1 min").pick(column: 0, value: "2 min")
+        XCTAssertTrue(app.find(value: "2 min").isVisible)
+
+        backButtonTap()
+    }
```

Comment:
This test was tricky since the rig doesn't have a picker with an a11y id. I couldn't find a way to find the picker, like we do in our app, so I added finding by value, and picking when you are on the pickerWheel.

Reply:
also looks like it fails on CI. do we want to have our own test app?

Reply:
CI was working, the app is installed as part of the workflow.

If you want to make an app that may be easier.